src/skeleton/error.rs

```diff
@@ -0,0 +1,37 @@
+use serialize::hex::FromHexError;
+
+/// Error that can happen while calculating an animation.
+pub enum SkeletonError {
+    /// The requested bone was not found.
+    BoneNotFound(String),
+
+    /// The requested slot was not found.
+    SlotNotFound(String),
+
+    /// The requested slot was not found.
+    SkinNotFound(String),
+
+    /// The requested slot was not found.
+    InvalidColor(String),
+
+    /// The requested animation was not found.
+    AnimationNotFound(String),
+}
+
+impl From<SkeletonError> for String {
```

Comment:
`SkeletonError` should implement `Debug` (with a #[derive]), `Error` and `Display` (by calling `Error::description`).
But not `Into<String>`.